### PR TITLE
Main index name

### PR DESCRIPTION
**Pull request by juliareichel**
changed home.html back to index.html

### CODE REVIEW

File: portfolio/src/main/webapp/activities.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    The Silent Sister</a>- Diane Chamberlain </li>
+              <li><a href="https://www.amazon.com/Fire-Sparkling-Julianne-MacLean-ebook/dp/B07L4PL653/ref=sr_1_1?crid=2XGSEFBPM5S2Y&dchild=1&keywords=a+fire+sparkling&qid=1590606516&s=digital-text&sprefix=a+fire+sp%2Cdigital-text%2C236&sr=1-1" target="_blank">
+                    A Fire Sparkling</a>- Julianne MacLean </li>
+              <li><a href="https://www.amazon.com/This-Tender-Land-William-Krueger-ebook/dp/B010MHAEGA/ref=sr_1_1?dchild=1&keywords=a+tender+land&qid=1590606634&s=digital-text&sr=1-10" target="_blank"> 
+                    This Tender Land</a>- William Kent Krueger </li>
+              <li><a href="https://www.amazon.com/Five-Years-Novel-Rebecca-Serle-ebook/dp/B07TFB5VWB/ref=sr_1_1?dchild=1&keywords=in+five+years&qid=1590606658&s=books&sr=1-1" target="_blank"> 
+                    In Five Years</a>- Rebecca Serle </li>
+              <li><a href="https://www.amazon.com/Bad-Blood-Secrets-Silicon-Startup-ebook/dp/B078VW3VM7/ref=sr_1_1?dchild=1&keywords=bad+blood&qid=1590606678&s=digital-text&sr=1-1" target="_blank"> 
+                    Bad Blood</a>- John Carreyrou </li>
```
Comment (damargulis) [inline portfolio/src/main/webapp/activities.html:58]: Loved this book!

File: portfolio/src/main/webapp/destinations.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                <li><a class="nav_item" href="facts.html">Facts</a></li>
+            </ul>
+        </nav><br>
+        <div id="page_title">
+            <div class="header">Julia Reichel's Portfolio</div>
+        </div>
+
+        <h2 id="vacation_header">Dream Vacation Spots</h2>
+        <div class="vacation_spots">
+          <div class = "destination">
```
Comment (damargulis) [inline portfolio/src/main/webapp/destinations.html:26]: nit.
remove the spaces around the =
(throughout the file)

File: portfolio/src/main/webapp/activities.html (hunk 1 above)
Comment (juliareichel) [reply portfolio/src/main/webapp/activities.html:58]: I am in the middle of it!

File: portfolio/pom.xml [hunk 3]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```
Comment (Kaleab470) [inline portfolio/pom.xml:37]: Consider removing the TODO

Comment (juliareichel) [reply portfolio/pom.xml:37]: good call, deleted it.

**Pull request merged**